core/pouch/migrations.js

```diff
@@ -197,6 +197,25 @@ const migrations /*: Migration[] */ = [
         return doc
       })
     }
+  },
+  {
+    baseSchemaVersion: 7,
+    targetSchemaVersion: 8,
+    description: 'Set all files executable attribute',
+    affectedDocs: (docs /*: Metadata[] */) /*: Metadata[] */ => {
+      return docs.filter(
+        doc => doc.docType === 'file' && doc.executable == null
```

Comment:
You are sure that we want to compare with `null`, and not with `undefined`?

Reply:
Using `==` both `null` and `undefined` values will return true here. I could specify `undefined` instead but if by any chance we do have some `null` values, I want to change them to false as well.